vault/identity_store.go

```diff
@@ -491,3 +503,17 @@ func (i *IdentityStore) CreateOrFetchEntity(ctx context.Context, alias *logical.
 
 	return entity, nil
 }
+
+// findAliasChanged searches an entity for changed alias metadata.
+//
+// If a match is found, the changed alias's index is returned. If no alias
+// names match or no metadata is different, -1 is returned.
+func findAliasChanged(entity *identity.Entity, alias *logical.Alias) int {
```

Comment:
Can we rename this function as `changedAliasIndex`?

Reply:
Yes.